tests/test_percy_snapshot.py

```diff
@@ -0,0 +1,82 @@
+import base64
+import os
+import time
+
+from .IntegrationTests import IntegrationTests
+import dash
+import dash_html_components as html
+import dash_core_components as dcc
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class Tests(IntegrationTests):
+    """
+    In order to render snapshots, Percy collects the DOM of the project and
+    uses a custom rendering method, different from Selenium. Therefore, it
+    is unable to render Canvas elements, so can't render Cytoscape charts
+    directly.
+
+    Instead, we use Selenium webdrivers to automatically screenshot each of
+    the apps being tested in test_usage.py, display them in a simple
+    Dash app, and use Percy to take a snapshot for CVI.
+    """
+
+    def test_usage(self):
+        def encode(name):
+            path = os.path.join(
+                os.path.dirname(__file__),
+                'screenshots',
+                name
+            )
+
+            with open(path, 'rb') as image_file:
+                encoded_string = base64.b64encode(image_file.read())
+            return "data:image/png;base64," + encoded_string.decode('ascii')
+
+        # Define the app
+        app = dash.Dash(__name__)
```

Comment:
@chriddyp check out this multi-route app. Here I Assign the url path to return the image it represent. For example, to return "usage.png", you can visit localhost/usage.png.